Makefile

```diff
@@ -62,7 +62,7 @@ YEAR_MONTH = $(shell date -u +'%Y%m')
 YEAR_MONTH_DAY = $(shell date -u +'%Y%m%d')
 MAJOR_MINOR_VERSION = $(shell echo $(BASE_VERSION) | cut -d '.' -f1).$(shell echo $(BASE_VERSION) | cut -d '.' -f2)
 PROTOC_VERSION = 3.10.1
-HELM_VERSION = 3.0.0
+HELM_VERSION = 3.8.0
```

Comment:
I'll add a commit message missing that you've also updated the minimum helm version

Reply:
never mind. you have that already